caffe2/ideep/operators/conv_op.cc

```diff
@@ -171,6 +171,58 @@ class IDEEPConvGradientOp final : public IDEEPConvPoolOpBase {
   OUTPUT_TAGS(FILTER_GRAD, BIAS_OR_INPUT_GRAD, INPUT_GRAD);
 };
 
+class IDEEPConvGradientOp final : public IDEEPConvPoolOpBase {
```

Comment:
This one is really weird. This part should be removed all together.

Reply:
Just checked, this part is still in fbcode master